### PR TITLE
Issue 895

### PR DESCRIPTION
**Pull request by PkHutch**
Updates the README.md to a more general guide on Courseography with some formatting changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,90 @@
...
+2. Navigate to `http://localhost:8000/graph` in your browser
+
+
+Contributors
+--------------------------------------
 
 This project would not exist without the contributions of many students in the Department of Computer Science. In alphabetical order, our contributors are:
 
-Alex Baluta, Alexander Biggs, Kelly Bell, Christina Chen, Eugene Cheung, Spencer Elliott, Ryan Fan, Christian Garcia, Ross Gatih, Philip Kukulak, Tamara Lipowski, Lydia Liu, Jahnavi Matholia, Christine Murad, San Shaftoe, Ian Stewart-Binks
+Baluta, Alex
```
Comment (PkHutch) [inline README.md]: Commit removed.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,90 @@
...
+
+##### Other Files
+*You can also do these two steps manually.*
+
+1. Generate app/DevelopmentConfig.hs to app/Config.hs with `$ cp app/DevelopmentConfig.hs app/Config.hs`
+2. Create the db folder with `$ mkdir db`
 
-Learn more about setting up and installing Courseography on your local machine for development [here](https://github.com/Courseography/courseography/wiki/Installing-Courseography).
+#### Installing
+1. Install GHC 7.10.3 with `$ cp app/Development`
```
Comment (PkHutch) [inline README.md]: See commit f3e14ec for fix.

File: README.md [hunk 3]
```diff
@@ -1,33 +1,82 @@
...
+Powered by [Haskell](https://www.haskell.org/), Courseography was started in late 2013 by [David Liu](http://www.cs.toronto.edu/~david/). However, it wasn't until he recruited [Ian Stewart-Binks](http://www.cs.toronto.edu/~iansb/) to the project that things really got rolling. Though the past two years have really seen our tools take off within the CS student body, there's still a long way for us to go. Our current projects include moving the front-end of the application over to [React](https://facebook.github.io/react/), unifying the graph viewing and drawing tools, and improved exporting and report generation.
+
 
-### Get Involved
+Getting Involved
+--------------------------------------
 
-See [CONTRIBUTING.md](https://github.com/Courseography/courseography/blob/master/CONTRIBUTING.md).
+Click [here](/CONTRIBUTING.md) if you would like to get involved.
```
Comment (PkHutch) [inline README.md]: Understood!

Comment (PkHutch) [inline README.md]: Should be reverted, see commit 2f8e3a33aab155482de55896d3453cce770df74c.

**Comment by david-yz-liu** [comment]
Thanks, @PkHutch! The only change I made was to remove the instruction to donwload the GitHub desktop client, as familiarity with Git should be a requirement for skipping the tutorial.

**Pull request merged**